pkg/storaged/stratis/pool.jsx

```diff
@@ -64,6 +65,197 @@ const _ = cockpit.gettext;
 const fsys_min_size = 512 * 1024 * 1024;
 const fsys_round_size = 1024 * 1024;
 
+/* Abstractions over the r6 and r8 API revisions.
+ */
+
+function get_key_descriptions(pool) {
+    const result = [];
+
+    if (!pool.Encrypted)
+        return result;
+
+    if (client.stratis_interface_revision < 8) {
+        const val = pool.KeyDescription;
+        if (val[0] && val[1][0])
+            result.push({ slot: null, keydesc: val[1][1] });
+    } else {
+        let val = pool.KeyDescriptions;
+        if ("t" in val)
+            val = val.v; // XXX - change notifications drop the variant wrapping for some reason.
```

Comment:
TODO @mvollmer: Create a GH issue for this and link here

Reply:
Done: https://github.com/stratis-storage/stratisd/issues/3895